src/app/theme.ts

```diff
@@ -2,11 +2,15 @@ import { createTheme } from '@mui/material/styles';
 
 export const theme = createTheme({
     palette: {
+        // for buttons to look like wikipedia links
         primary: {
-            main: '#0052cc',
+            main: '#3366cc',
```

Comment:
This can be only problem that I see in this code. There will be conflict with our color palette.

Reply:
ok, lets merge yours first and I will handle the conflicts :)